examples/platforms/posix/flash-windows-stubs.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
@nibanks , The current Posix flash driver can't be built on Windows platform, so I add this stub to pass the appveyor check temporarily, please let me know if you have any suggestions, thanks.